src/complex.cr

```diff
@@ -1,5 +1,14 @@
+# A complex number is a number represented in the form a + bi. In this form, a and b are real numbers, and i is an imaginary number such as i² = -1. The a is the real part of the number, and the b is the imaginary part of the number.
```

Comment:
Perhaps this can be wrapped at 78/80 columns? Will make more easy to spot issues when using the PR review functionality.

Thank you.